frontend/src/components/BalancePresentation.js

```diff
@@ -5,13 +5,18 @@ import "../styles/BalancePresentation.css";
 class BalancePresentation extends React.Component {
   constructor(props) {
     super(props);
+
     this.state = {
-      salary: 5000,
+      salary: this.props.salary,
```

Comment:
We should rather remove salary from state, we don't need to save it in the BalancePresentation state since we're now getting it in as a prop.

Reply:
changes done.

Reply:
9a3fedac0085772f77a7003f22ef18776a761afa OK